src/Microsoft.TestPlatform.Client/Discovery/DiscoveryRequest.cs

```diff
@@ -26,6 +26,7 @@ namespace Microsoft.VisualStudio.TestPlatform.Client.Discovery;
 public sealed class DiscoveryRequest : IDiscoveryRequest, ITestDiscoveryEventsHandler2
 {
     private readonly IDataSerializer _dataSerializer;
+    internal /* for testing purposes */ readonly ProtocolConfig _protocolConfig = Constants.DefaultProtocolConfig;
```

Comment:
I am not sure whether it's better to:
1. Have this flow from ctor (might impact public api - I haven't checked)
2. Use directly `Constants.DefaultProtocolConfig` in the check
3. Current solution
4. Something else? Please let me know

Reply:
I would prefer directly using `Constants.DefaultProtocolConfig` instead of redeclaring it in files.

Reply:
Done